src/services/laodeai/handlers/gist.js

```diff
@@ -1,4 +1,4 @@
-import { sanitize } from '../../utils/sanitize.js';
+import { sanitize } from '../../../utils/sanitize.js';
```

Comment:
nodejs bisa dikasih alias ga sih? ngeselin juga lama lama numpuk gini

Reply:
bisa, pake jsconfig.json (https://code.visualstudio.com/docs/languages/jsconfig) tapi setauku kalo mau kayak gitu harus di build dulu, pake webpack atau sejenisnya. which I think a bit too much for the project.

Reply:
yowes lah, gini aja

Reply:
after some googling, this might work https://nodejs.org/api/packages.html#packages_subpath_imports